components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -344,7 +346,7 @@ public static void usage() {
             + "  $ %s -s localhost -u username -w password -d 50 --debug ALL foo.tiff\n"
             + "\n"
             + "For additional information, see:\n"
-            + "http://www.openmicroscopy.org/site/support/omero5/users/command-line-import.html\n"
+            + "http://www.openmicroscopy.org/site/support/omero5.1/users/cli/import.html\n"
```

Comment:
Hmmm.... use of 5.1 I find slightly worrying, unless we have a reminder to `s/5.1/5.2/` post-fork.